components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -921,8 +922,10 @@ void RewardsServiceImpl::OnWalletInitialized(ledger::Result result) {
 
     // Record P3A:
     RecordWalletBalanceP3A(true, 0);
-    const bool ads_enabled =
-        profile_->GetPrefs()->GetBoolean(brave_ads::prefs::kEnabled);
+    bool ads_enabled = false;
```

Comment:
it doesn't seem like there's any reason to collect information about ads enabled here if it's not enabled in the build. I would leave the block as-is and include `RecordAdsState` in it